### PR TITLE
Bring in nhsuk-frontend ESM syntax JS files to enable compilation to CJS

### PR DESCRIPTION
**Pull request by jakeb-nhs**

### CODE REVIEW

File: src/components/content-presentation/tabs/Tabs.tsx [hunk 1]
```diff
@@ -55,7 +55,8 @@ interface Tabs extends FC<TabsProps> {
 
 const Tabs: Tabs = ({ className, children, ...rest }) => {
   useEffect(() => {
-    TabsJs.default ? TabsJs.default() : TabsJs();
+    console.warn('Running tabs useeffect');
```
Comment (JoshuaBates-NHS) [inline src/components/content-presentation/tabs/Tabs.tsx:58]: do you still want this here?

File: src/components/content-presentation/tabs/Tabs.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import classNames from 'classnames';
 import React, { FC, HTMLAttributes, useEffect } from 'react';
 import HeadingLevel, { HeadingLevelType } from '@util/HeadingLevel';
-import TabsJs from 'nhsuk-frontend/packages/components/tabs/tabs';
+import TabsJs from './../../../scripts/tabs';
```
Comment (JoshuaBates-NHS) [inline src/components/content-presentation/tabs/Tabs.tsx:5]: could we put these under an alias

Comment (jakeb-nhs) [reply src/components/content-presentation/tabs/Tabs.tsx:5]: I did try this but rollup wasn't able to resolve the files when using an alias, I'll give it a fresh go and see if I've missed anything

File: src/components/content-presentation/tabs/Tabs.tsx (hunk 1 above)
Comment (jakeb-nhs) [reply src/components/content-presentation/tabs/Tabs.tsx:58]: nope good spot

**Review by JoshuaBates-NHS — APPROVED** [review]

**Pull request merged**